l1-contracts/contracts/state-transition/ValidatorTimelock.sol

```diff
@@ -180,15 +181,19 @@ contract ValidatorTimelock is IExecutor, Ownable2Step {
 
     /// @dev Check that batches were committed at least X time ago and
     /// make a call to the hyperchain diamond contract with the same calldata.
-    function executeBatches(StoredBatchInfo[] calldata _newBatchesData) external onlyValidator(ERA_CHAIN_ID) {
-        _executeBatchesInner(ERA_CHAIN_ID, _newBatchesData);
+    function executeBatches(
+        StoredBatchInfo[] calldata _batchesData,
+        PriorityOpsBatchInfo[] calldata
+    ) external onlyValidator(ERA_CHAIN_ID) {
```

Comment:
We should remove `commitBatches`/`proveBatches `/`executeBatches ` functions since they are not used anymore on Era while available only for Era.

Reply:
We'll resolve in the kl/sync-layer-reorg first and then merge here